cves/CVE-2018-13304.yml

```diff
@@ -135,7 +128,13 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    This vulnerability was caught shortly after the support for 8-bit 
+    and Simple Studio Profile where added, and a few other bugs/vulnerabilities
+    where fixed before this specific problem was addressed. The first few fixes
```

Comment:
Good answer!

This is a simple ask, but could you explain what a `profile` is somewhere?

Reply:
Now in description